tasks/systemd.yml

```diff
@@ -0,0 +1,24 @@
+---
+- name: Download systemd service files from gitlab
+  get_url:
+    url:  "https://gitlab.com/gitlab-org/gitlab-recipes/raw/master/init/systemd/{{ item }}"
+    dest: "/etc/systemd/system/{{ item }}"
+  with_items:
+    - gitlab-sidekiq.service
+    - gitlab-unicorn.service
+    - gitlab-workhorse.service
+    - gitlab-mailroom.service
+    - gitlab-gitaly.service
+
+- name: Enable and start GitLab services
+  systemd: 
+    state: started
+    enabled: yes
+    daemon_reload: yes
```

Comment:
Please just do the damon-reload when a unit was changed

Reply:
Fixed